hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/TestHiveSyncTool.java

```diff
@@ -641,6 +643,26 @@ public void testSyncWithSchema(String syncMode, String enablePushDown) throws Ex
         "The last commit that was synced should be updated in the TBLPROPERTIES");
   }
 
+  @ParameterizedTest
+  @MethodSource("syncModeAndEnablePushDown")
+  public void testSyncTimestamp(String syncMode, String enablePushDown) throws Exception {
+    hiveSyncProps.setProperty(HIVE_SYNC_MODE.key(), syncMode);
```

Comment:
Can we add a test case similar with `testSchemaConvertTimestampMicros`, there is no need to add avro schema files.